advanced/databases/mysql-is/requirements.yaml

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 dependencies:
   - name: mysql
-    version: "8.2.3"
-    repository: "https://charts.bitnami.com/bitnami"
+    version: "1.6.9"
+    repository: "https://helm.wso2.com"
```

Comment:
@chirangaalwis the existing value never starts and the chart doesn't get to the IS deploy. this value was found on the 5.11.x branch and worked so I went with it.

Reply:
I think the given chart seems like a valid version as per official Artifact Hub [link](https://artifacthub.io/packages/helm/bitnami/mysql/8.2.3). @Aaquiff can you please verify this since you tested this out?

Reply:
I pulled this fromt he 5.11.x branch so this change will go away when I change the merge destination.